CHANGELOG.md

```diff
@@ -1,13 +1,19 @@
-## vUNRELEASED
+## v1.1.0
+
+This release is the first incremental release since 1.0. It includes bugfixes and some UX improvements.
 
 #### New features and UX changes
 
 - Add support for non-numerical UID/GID as specified in the appc spec ([#2159](https://github.com/coreos/rkt/pull/2159)).
```

Comment:
Another possibility:

- rkt can now start apps as the user and group specified in the [image manifest](https://github.com/appc/spec/blob/master/spec/aci.md#image-manifest-schema) with three different possibles formats: a numeric UID/GID, a username and group name referring to the ACI's /etc/passwd and /etc/group, or a file path in the ACI whose owner will determine the UID/GID ([#2159](https://github.com/coreos/rkt/pull/2159)).

What do you think?